exo-add/features/steps/steps-when.ls

```diff
@@ -42,6 +42,19 @@ module.exports = ->
       ..on 'ended', done
 
 
+  @When /^trying to run "([^"]*)" in this application's directory$/, timeout: 10_000, (command, done) ->
+    args = command.split ' '
+    args[0] = path.join process.cwd!, 'bin', args[0]
+    if process.platform is 'win32'
+      args[0] += '.cmd'
+    @process = new ObservableProcess(args,
+                                     cwd: @app-dir,
+                                     stdout: dim-console.process.stdout
+                                     stderr: dim-console.process.stderr)
+      ..on 'ended', ~>
+        | @process.exit-code > 0  =>  done!
```

Comment:
What happens if the exit code is 0? It should end with an error, not time out after 10 sec.